src/map-interactions.react.js

```diff
@@ -325,7 +326,9 @@ export default class Interactions extends Component {
   render() {
     return (
       <div
-        ref="container"
+        ref={container => {
```

Comment:
Yeah I wonder if inline definition is what's causing the callback to be called every frame in v3. I will investigate.